src/less/components/button.less

```diff
@@ -204,120 +202,103 @@
-    //                 }
-    //             }
-    //         }
-    //     }
-
-    //     &.btn-sm,
-    //     &.btn-xs {
-    //         .loader {
-    //             .btn-loader-variant(0.875rem; 0.2rem; 0.125rem; 0.125rem; 0.875rem; 0.5rem);
-
-    //             .loader-icon {
-    //                 li:first-child {
-    //                     animation: loadbarsbtnsm 0.6s cubic-bezier(0.645, 0.045, 0.355, 1) infinite 0s normal;
-    //                 }
-
-    //                 li:nth-child(2) {
-    //                     animation: loadbarsbtnsm 0.6s ease-in-out infinite -0.2s normal;
-    //                 }
-
-    //                 li:nth-child(3) {
-    //                     animation: loadbarsbtnsm 0.6s ease-in-out infinite -0.4s normal;
-    //                 }
-    //             }
-    //         }
-    //     }
-
-    //     &.btn-guiding {
-    //         .loader {
-    //             .loader-color-variant(@btn-guiding-color);
-    //         }
-    //     }
-
-    //     &.btn-link {
-    //         .loader {
-    //             .loader-color-variant(@blue);
-    //         }
-    //     }
-
-    //     &.btn-outline-executive {
-    //         .loader {
-    //             .loader-color-variant(@btn-executive-outline-color);
-    //         }
-
-    //         &:hover {
-    //             .loader-color-variant(@btn-executive-color);
-    //         }
-    //     }
-
-    //     &.btn-outline-guiding {
-    //         .loader {
-    //             .loader-color-variant(@btn-guiding-outline-color);
-    //         }
-
-    //         &:hover {
-    //             .loader-color-variant(@btn-guiding-color);
-    //         }
-    //     }
-
-    //     &.btn-outline-destructive {
-    //         .loader {
-    //             .loader-color-variant(@btn-destructive-outline-color);
-    //         }
-
-    //         &:hover {
-    //             .loader-color-variant(@btn-destructive-color);
-    //         }
-    //     }
-    // }
+    &.loading {
+        &:after {
+            content: "";
+            border-radius: 50%;
+            margin-left: 0.5rem;
+            border: 0.15rem solid;
+            width: 1rem;
+            height: 1rem;
+            animation: loader-spin 0.8s linear infinite;
+        }
+
+        &.btn-executive,
+        &.btn-guiding {
+            &:after {
+                border-color: hsla(0, 0%, 100%, 0.5);
+                border-left-color: @white;
+            }
+        }
+
+        &.btn-destructive {
+            &:after {
+                border-color: rgba(47, 36, 36, 0.4);
+                border-left-color: @dark-brown;
+            }
+
+            &[disabled] {
+                &:after {
+                    border-color: rgba(47, 36, 36, 0.2);
+                    border-left-color: @destructive-disabled-color;
+                }
+            }
+        }
+
+        &.btn-outline-executive {
```

Comment:
`btn-outline btn-executive` easier to remember (order does not matter) and it will scale better if we add another modifier class.